### ISSUE TITLE
Help Text - Provide 2 examples in plain English (eg.: Gini and Responsiveness) of what the metric is and how it is calculated 

### ISSUE THREAD

**Issue by matextrem** [protofire/maker-governance-dashboard#332]
The subgraph query does not fully explain it and it is not suitable for non-tech people

**Comment by matextrem** [comment]
See this document for more information https://docs.google.com/document/d/1ru_pHyZXxQ4YZ2_nkbHXm9eUdnNn_OYSUYv-rI_knBI/edit

**Comment by tatyanakurinskaya** [comment]
1. Staked MKR Info: 
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner

2. Number of Voters Info:
- update window title to 'Number of Voters vs Total MKR Staked Info'
- 'MKR Registry Graph' link points to 'MakerDAO Governance Graph' --> either update text on link or link

3. Total Votes Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

4. Voting MKR Gini Coefficient Info:
- missing some text, it should be:
Allows us to keep track of how ‘fair’ the distribution of voting tokens is in the MKR governance ecosystem.

First of all, we get all the actions related to free and lock MKR during a specified period of time (Last year, last month, etc). After that, we loop over each period and calculate for each sender the total MKR locked in the Chief (using the lock and free events) . Finally, we use those values to get the Gini coefficient.

- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

5. MKR Activeness Breakdown Info:
- update 'Maker Governance Graph' to 'MakerDAO Governance Graph'

6. Votes - MKR Responsiveness Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
- missing the ending, text should be:
The more responsive MKR tokens are, the safer the system is. It gives a good ballpark estimate for how much MKR we can expect to see voting in executive votes and how quickly that MKR will vote.

First of all, we get all the voting actions for each executive vote. Then, we loop over the periods of time (30-Day bucket) and we group those happening at the same day. Finally, we sum the amount of MKR locked in the Chief at the moment the votes occured, grouped by day.

7. Polls - MKR Responsiveness Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner

8. Top Voters Info:
- update 'MakerDao Governance Graph - Executives' to 'MakerDAO Governance Graph - Executives'
- update 'MakerDao Governance Graph - Polls' to 'MakerDAO Governance Graph - Polls'
- links 'MakerDao Governance Graph - Executives' and 'MakerDao Governance Graph - Polls' are concatenated. ---> place 'MakerDao Governance Graph - Polls' at the right bottom corner

9. Top Executives Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

10. Uncast Executives Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

11. Executive Time to Pass Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

12. MKR Distribution By Executive Info:
- update 'MakerDao Governance Graph - Governance Info' to 'MakerDAO Governance Graph - Governance Info'
- update 'MakerDao Governance Graph - Executives' to 'MakerDAO Governance Graph - Executives'
- links are concatenated. ---> place 'MakerDao Governance Graph - Governance Info' at the right bottom corner

13. Most Voted Polls Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

14. Recent Polls Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

15. MKR Staked Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

16.Number of Voters Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

17. Approvals by Address Size Info
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

18. Top Supporters Info
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

19. Vs Current Hat Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'

20. Vote Count By Option Info:
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner

21. Voter Per Option Info
- update title in widget to 'Votes Per Option' and modal to 'Votes Per Option Info'
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner
- text should be : 
Shows the current or final distribution of voting addresses across the options available in this poll. This gives an intuitive visual depiction of the current or final vote results in terms of addresses.

22. MKR Voter Per Option
- update title in widget to 'MKR Votes Per Option' and modal to 'MKR Votes Per Option Info'
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner
- text should be : 
Shows the current or final distribution of MKR voting across the options available in this poll. This gives an intuitive visual depiction of the current or final vote results in terms of MKR.

23. MKR Count By Option Info
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner
- 'MKR Registry Graph' points to 'MakerDAO Governance Graph', 'MakerDAO Governance Graph' links to 'MKR Registry Graph'

24. Top Voters Info
- update 'MakerDao Governance Graph' to 'MakerDAO Governance Graph'
-  links 'MakerDao Governance Graph' and 'MKR Registry Graph' are concatenated. ---> place 'MKR Registry Graph' at the right bottom corner

**Comment by matextrem** [comment]
Last paragraphs in Gini coefficient and Votes Responsiveness were added by us as an example for MakerDAO. They shouldn't be added now.
